doc/source/release/1.14.0-notes.rst

```diff
@@ -334,7 +334,7 @@ eliminating their use internally and two new C-API functions,
 
 have been added together with a complementary flag,
 ``NPY_ARRAY_WRITEBACKIFCOPY``. Using the new functionality also requires that
-some flags be changed when new arrays are created, to wit:
+some flags be changed when new arrays are created:
```

Comment:
Please leave these files out.

Reply:
skipping and dropping manual 21d95b25f5ae2c09e5cc6f55bbcf785aa957c8dd